tests/interlocked_win32_ut/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(theseTestsName interlocked_win32_ut)
+
+set(${theseTestsName}_test_files
+${theseTestsName}.c
+)
+
+set(${theseTestsName}_c_files
+mock_interlocked.c
+)
+
+set(${theseTestsName}_h_files
+../../inc/azure_c_util/interlocked.h
+)
+
+build_c_tests(${theseTestsName} ON "tests/azure_c_util_tests" ADDITIONAL_LIBS synchronization azure_c_util_reals)
```

Comment:
>azure_c_util_reals [](start = 94, length = 18)

I did not notice any function being used from azure_c_util_reals.

What happens when you remove this? Maybe replace it with logging lib...